features/admin.applications.v1/components/api-authorization/subscribed-api-resources.tsx

```diff
@@ -144,6 +147,13 @@ export const SubscribedAPIResources: FunctionComponent<SubscribedAPIResourcesPro
         useState<AuthorizedAPIListItemInterface[]>(null);
     const [ copyScopesValue, setCopyScopesValue ] = useState<string>(null);
     const [ m2mApplication, setM2MApplication ] = useState<boolean>(false);
+    const dispatch: Dispatch = useDispatch();
```

Comment:
Keep useStates and hooks separated for readability. Prefer moving this line under line 142.
```suggestion

    const dispatch: Dispatch = useDispatch();
```